src/voice-pe/troubleshooting/no-voice-response.md

```diff
@@ -16,11 +16,17 @@ The voice response is generated in Home Assistant by one of our supported text-t
 
 ## Resolution
 
-Check the Home Assistant URL.
+Make sure you can reach Home Assistant via the local Home Assistant URL (from inside your network, not using the external URL you use to access Home Assistant from outside your network).
```

Comment:
Think it should be
> Make sure the **voice assistant device** can reach Home Assistant via the local Home Assistant URL

Reply:
Also good to note that the voice assistant device will reject invalid SSL certificates